content/zh-cn/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -118,7 +118,7 @@ And see the frontend one you created:
 -->
 并看到你所创建的前端：
 
-```
+```shell
 NAME       DESIRED   CURRENT   READY   AGE
 frontend   3         3         3       6s
 ```
```

Comment:
This is not a shell script. We (previous contributor) are intentionally remove the wrong syntax lang tag.